ipmininet/tests/test_sshd.py

```diff
@@ -0,0 +1,35 @@
+"""This module tests the SSH daemon"""
+import os
+import time
+
+from ipmininet.clean import cleanup
+from ipmininet.examples.sshd import SSHTopo
+from ipmininet.ipnet import IPNet
+from . import require_root
+
+
+@require_root
+def test_sshd_example():
+    try:
+        net = IPNet(topo=SSHTopo())
+        net.start()
+
+        ssh_key = None
+        with open("/tmp/sshd_r2.cfg") as fileobj:
```

Comment:
We can definitely improve the whole ergonomics of the SSH key handling here. Likely though an helper in the config object.

Reply:
I could just import KEYFILE in ipmininet.router.config.sshd. Isn't it sufficient?
This code is needed because I want to check the content written in the configuration.